app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  respond_to :json
+  # before_action :configure_sign_up_params, only: [:create]
+  # before_action :configure_account_update_params, only: [:update]
+
+  # GET /resource/sign_up
+  # def new
+  #   super
+  # end
+
+  # bug in devise for rails 7 https://github.com/heartcombo/devise/issues/5443
+  def sign_up(resource_name, resource)
+    sign_in(resource_name, resource, store: false)
+  end
```

Comment:
Just FYI: eventually we'll let people sign up but for now they are invite only, sorry should have mentioned this.

Reply:
We can jsut keep signup but keep it hidden for now too

Reply:
ok, so we they'll receive an email for sign up? then their password will be created? so i'm guessing the user record is created before the password is?

Reply:
We'll create all the user objects with a spreadsheet.   i think without password.  they get invited via email with a special token that authenticates them automatically (without password).   that's just token auth, there's a devise plugin but also super easy to just code up.    and we can support normal email/password auth of course too.